docs/_sidebar.md

```diff
@@ -78,6 +78,7 @@
   - [FaaS design](en/design/faas/faas-poc-design.md)
 - Contributing
   - [Document Contribution Guide](en/development/contributing-doc.md)
+    - [Automate testing of Quickstart documentation with tools](en/development/test-quickstart.md)
```

Comment:
We can make the directory structure same with [chinese doc](https://mosn.io/layotto/#/zh/development/contributing-doc) :
![image](https://user-images.githubusercontent.com/26001097/172662614-d771395e-bab0-4d57-a6e6-1015f9315a57.png)

Reply:
update